browser-test/src/static_text.test.ts

```diff
@@ -41,6 +58,15 @@ describe('Static text question for applicant flow', () => {
     await applicantQuestions.seeStaticQuestion(staticText)
   })
 
+  it('displays the accordion', async () => {
```

Comment:
suggestion (non-blocking): Sorry I missed this before, but you could also combine this test with `validate screenshot when accordion closed` which might be nice. One test for accordion open, one test for accordion closed.

Reply:
Sure, will change that.